frontend/lib/search/result_list.dart

```diff
@@ -0,0 +1,83 @@
+import 'dart:math';
+
+import 'package:ehrenamtskarte/graphql/graphql_api.graphql.dart';
+import 'package:ehrenamtskarte/map/detail/detail_view.dart';
+import 'package:ehrenamtskarte/widgets/error_message.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:graphql_flutter/graphql_flutter.dart';
+
+class ResultList extends StatelessWidget {
+  final String _searchText;
+  final List<int> _searchCategories;
+
+  ResultList(this._searchText, this._searchCategories);
```

Comment:
```suggestion
  ResultList(this._searchText, this._searchCategories, {Key key}) : super(key: key);
```

Reply:
Why is this needed?

Reply:
I guess this is standard in flutter... Damn that boilerplate :/

Reply:
It is probably not needed at the moment, but good style to still add it so that you do not have to touch this file later on when you need a key.
The key is necessary for example when you replace a widget in the tree by an other of the same type, to allow Flutter to detect the change by comparing the key.